src/apply.jl

```diff
@@ -199,8 +199,8 @@ function ITensors.apply(
     end
 
     if normalize
-      normalize!(ψv1)
-      normalize!(ψv2)
+      ψv1 ./= norm(ψv1)
+      ψv2 ./= norm(ψv2)
```

Comment:
This is still an in-place operation.

If you want to make it out-of-place use:
```julia
      ψv1 /= norm(ψv1)
      ψv2 /= norm(ψv2)
```
(without the `.`).